README.md

```diff
@@ -1,12 +1,94 @@
 <h2 align="center"> AMAZING!!😮:star_struck:RIGHT?</h2>
 
+## Demo
+Want to know more about EduHelp. Just click the Image Below :point_down:
+<p align="center">
+  <a href="https://youtu.be/qgMU8F1QX5g"><img src="https://img.youtube.com/vi/qgMU8F1QX5g/maxresdefault.jpg" width=500 height=200></img></a>
+</p>
+
+## What Do We Think Of Implementation ?
+ - There are lot of Features which needs to be implemented for this project. To know what are those please click :point_right:[Here](https://docs.google.com/document/d/1NgP7kcJPSz0XaIq51hJkgtHk4Pm7J58N76-_0py2DTc/edit?usp=sharing).
+ - We would love to add other features in this project. It will be greatly appreciated if you contribute to this project and we ensure, it will be given a higher rating, i.e. level1, level2, level3 based on the usefulness and how inspirational your idea is, here by idea I mean just the idea and the flow of Idea in a `.md` file. If you can write your code for the idea you suggested, we will deal it in separate PR’s and with different level tags.
+
+## Tech-Stacks 💻
+- For Website: `HTML`, `CSS`
+- For core functionalities: `Python`
+- For summarizer part: `NLP`
+- For different Speech audio being clustered: `ML`
+- `Electron.js` or `Tkinter` or any other Desktop App development stack.
+- For making all the functionalities available as an API: `Flask Backend`
+
+## How to get started?
+<h3 align="center">A beginner and don't know How and Where to start???</h3>
+<p align="center">
+   <img src="https://user-images.githubusercontent.com/68437435/111859661-70fa7b80-8968-11eb-809d-a8e9a7266917.gif" width=300 height=200>
+</p>
+Don't worry, You can refer to the following articles and also contact the Project Maintainers, in case you are stuck:
+
+- [Watch this video to get started, if you have no clue about open source](https://www.youtube.com/watch?v=c6b6B9oN4Vg)
+- [Forking a Repo](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+- [Cloning a Repo](https://help.github.com/en/desktop/contributing-to-projects/creating-a-pull-request)
+- [How to create a Pull Request](https://opensource.com/article/19/7/create-pull-request-github)
+- [Getting started with Git and GitHub](https://towardsdatascience.com/getting-started-with-git-and-github-6fcd0f2d4ac6)
+
+## How to Contribute?
```

Comment:
Don't capitalize the 1st letter of every word. Do it for only the 1st word. Rest all words should be in lower case.